### ISSUE TITLE
console noise in 3.15

### ISSUE THREAD

**Issue by kellyselden** [emberjs/data#6946]
I just created a new app and ran ember serve:

```
- building... [BroccoliRollup > rollup]Non-existent export 'default' is imported from -private\ts-interfaces\record-data.js
| building... [Babel: ember-inflector > applyPatches]'@ember-data/model/-private' is imported by -private\index.js, but could not be resolved – treating it as an external dependency
```

Apologies if this is tracked elsewhere.

**Comment by efx** [comment]
IIRC the discord conversation in #dev-ember-data:
<img width="816" alt="Screen Shot 2020-01-06 at 14 03 43" src="https://user-images.githubusercontent.com/472721/71841429-64547500-308d-11ea-8bef-41df3c3e361c.png">

**Comment by sandstrom** [comment]
I think this was fixed in 3.16 (https://github.com/emberjs/data/pull/6809)
